client/package.json

```diff
@@ -147,9 +144,7 @@
     "ci_route_load_tests": "node browser-tests/route-load-tests.js --browser chromium --sandbox false --concurrency 8",
     "ci_serve": "http-server -s -a 127.0.0.1",
     "serve-loopback": "http-server -a 127.0.0.1",
-    "serve": "http-server",
-    "postinstall": "npx patch-package",
-    "postci": "npx patch-package"
```

Comment:
Could drop the patch-package dependency (unless we use it for the more granular fix mentioned above).